src/pages/monetization/partner-program-api/index.mdx

```diff
@@ -2691,6 +2692,87 @@ curl --request GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+## Get Solution Licenses by status
+
+`GET /v2/partners/solutions/licenses/<status>` - get the list of licenses by status
+
+#### Headers
+
+- `Authorization` - **required** - `Bearer <YOUR_API_TOKEN>`
+
+#### Path Parameters
+
+- `<status>` - **required** - license status, one of: `paid`, `trial`, `expired`
+
+#### Query Parameters
+
+- `date_from` - **required** - get a report for licenses created from a given date
+- `date_to` - **required** - get a report for licenses created up to a given date
+- `limit` - an amount of licenses in response (min. 1, max. 15 - default)
```

Comment:
Is 15 the default value? I'm asking as it's not entirely clear to me here :D

Reply:
Yes, it's the default :) I changed it, hope it's more clear - https://github.com/livechat/livechat-public-docs/pull/1410/commits/ea0961a8ea444c96c9ebf419367b3626139c67dd#diff-1fdbc72100416ca7bcf4938c953d51e3a2520b5431ba09b1205cf0b177de152fR2715